### PR TITLE
[v6r17] dirac-install.py should fail when installation of lcgBundle has failed

### PR DESCRIPTION
**Pull request by iueda**
installExternals to return False when lcgBundles failed

BEGINRELEASENOTES
*Core
FIX: dirac-install.py to fail when installation of lcgBundle has failed

ENDRELEASENOTES

### CODE REVIEW

**Comment by atsareg** [comment]
Should it be just a warning ? In most cases the installation of LCG bindings fails for clients on platforms for which LCG software is not compiled. In this case some functionality subset is indeed not available but the client is functional for most cases. This is why the installation is declared successful even if the LCG bindings are not available

**Comment by iueda** [comment]
I was expecting such responses in the issue https://github.com/DIRACGrid/DIRAC/issues/3388 before my making this PR. Probably we should go to the issue for discussion?

**Comment by fstagni** [comment]
Looks OK to me.

File: Core/scripts/dirac-install.py [hunk 1]
```diff
@@ -1267,6 +1284,8 @@ def installExternals( releaseConfig ):
     #HACK: try to find a more elegant solution for the lcg bundles location
     if not downloadAndExtractTarball( tarsURL + "/../lcgBundles", "DIRAC-lcg", verString, False, cache = True ):
       logERROR( "Check that there is a release for your platform: DIRAC-lcg-%s" % verString )
+      logERROR( "If you do not need LCG utils, use the option --no-lcg-bundle to skip LCG Bundle installation." )
```
Comment (atsareg) [inline Core/scripts/dirac-install.py]: I would not make any economy on this error message. So, I suggest that this is a multiline message, something like:

The requested LCG software version %s for the local operating system is not found.
Please, check the availability of the LCG software bindings for you platform.

If you want to skip the installation of the LCG software, redo the installation 
with adding --no-lcg-bundle command line option.

**Pull request merged**